File: cantina/src/main/java/br/com/cadalunos/cantina/Entity/Aluno.java
package br.com.cadalunos.cantina.Entity;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Table(name = "aluno")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Aluno {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String nome;
    private String sexo;
    private int anoNascimento;

    @Column(nullable = false, unique = true, length = 20)
    private String matricula;

    private String turma;
}

File: frontend/CadAlunos/src/app/models/aluno.ts
export class Aluno {

  id!: number;
  nome!: string;
  sexo!: string;
  anoNascimento!: number;

  constructor(id: number, nome: string, sexo: string, anoNascimento: number) {
    this.id = id;
    this.nome = nome;
    this.sexo = sexo;
    this.anoNascimento = anoNascimento;
  }
}